client/components/App.js

```diff
@@ -219,6 +220,15 @@ class App extends Common {
           ).toNumber()
         );
       }
+      const depositLenght = await poolContract.getDepositsLength(
+        connectedWallet
+      );
+      for (let i = 0; i < depositLenght; i++) {
+        let deposit = await poolContract.getDepositByIndex(connectedWallet, i);
+        if (deposit.tokenType >= 5 && deposit.unlockedAt === 0) {
```

Comment:
Do not use >= 5. Set up the constant and use the constant. if now, whoever reads the code in the future has no clue of why there is a 5.